src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
-<h1>Welcome to Food Shop!</h1>
+<h1>Welcome to {{restaurantName}}</h1>
```

Comment:
Can we please move this line to the landing.component.html

Reason: what we put in app.component.html will appear at every page, whereas this welcome message we want to only appear in the landing page.

Reply:
This is completed.